.changes/amend

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python3
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from argparse import ArgumentParser
+
+from tool.amend import amend
+
+
+def main() -> None:
```

Comment:
Looking through our changelog, we have some instances where we've grouped multiple different PRs under the same changelog entry (like, "Fixed various issues with protocol tests"). We could do that because the changelog came only after all changes for the release were known. With this, changelog entries either come with each individual PR, or not at all (if you omit one for a PR). Given how it will now be certainly easier to just add a changelog entry for each PR and not worry about trying to combine them at the end, I don't mind this, but just something to note.

Reply:
You can still add them later, or amend them. This is why the associated pr list is a list